src/Interface/Tabs/PluginList.as

```diff
@@ -8,7 +8,9 @@ class PluginListTab : Tab
 	int m_total;
 	int m_page;
 	int m_pageCount;
+
 	array<PluginInfo@> m_plugins;
+	array<PluginInfo@> m_pluginsWithChangelogs;
```

Comment:
Why is this extra array required?

Reply:
That's there so CheckChangelogRequests() only checks in on plugins that have ongoing requests. I'm not sure if there's a better way to do it, but it's basically just an array of PluginInfo that have that request not fulfilled. I couldn't get it to play nice without something like this unfortunately.

Reply:
Can you not just check if the handle for a request is not null, or set some bool to true when it's doing the request?

Reply:
Yeah true, idk why I was so scared of iterating through the original array when I wrote this lol, I'll fix this once I get home

Reply:
fix is pushed! turns out i had that other array because i accidentally did a `return` instead of a `continue` in a for loop and the array was me jankily working my way around this bug without realizing it.

skill issue i know